python/sglang/srt/server_args.py

```diff
@@ -2185,6 +2201,12 @@ def check_server_args(self):
 
         # Check multi tokenizer
         assert self.tokenizer_worker_num > 0, "Tokenizer worker num must >= 1"
+        self.validate_buckets_rule(
+            "--prompt-tokens-buckets", self.prompt_tokens_buckets
+        )
+        self.validate_buckets_rule(
+            "--generation-tokens-buckets", self.generation_tokens_buckets
+        )
```

Comment:
The configuration options for these two parameters are slightly complex, so perhaps we should add a separate documentation page to explain them.